src/utils/helpers/list/useGetFilterDefinitions.ts

```diff
@@ -1,23 +1,29 @@
 import { DocumentNode } from 'graphql'
 import { DateTime, DurationObjectUnits } from 'luxon'
-import { today } from '../dateAndTime/parseDateRange'
+import { today } from '../../dateAndTime/parseDateRange'
 import {
   ApplicationOutcome,
   ApplicationStatus,
   AssignerAction,
   ReviewerAction,
-} from '../generated/graphql'
-import getApplicantFilterList from '../graphql/queries/applicationListFilters/getApplicantFilterList'
-import getAssignerFilterList from '../graphql/queries/applicationListFilters/getAssignerFilterList'
-import getOrganisationFilterList from '../graphql/queries/applicationListFilters/getOrganisationFilterList'
-import getReviewersFilterList from '../graphql/queries/applicationListFilters/getReviewersFilterList'
-import getStageFilterList from '../graphql/queries/applicationListFilters/getStageFilterList'
-import { FilterDefinitions, GetFilterListQuery, NamedDates } from '../types'
-import { useLanguageProvider } from '../../contexts/Localisation'
+} from '../../generated/graphql'
+import getApplicantFilterList from '../../graphql/queries/applicationListFilters/getApplicantFilterList'
+import getAssignerFilterList from '../../graphql/queries/applicationListFilters/getAssignerFilterList'
+import getOrganisationFilterList from '../../graphql/queries/applicationListFilters/getOrganisationFilterList'
+import getReviewersFilterList from '../../graphql/queries/applicationListFilters/getReviewersFilterList'
+import getStageFilterList from '../../graphql/queries/applicationListFilters/getStageFilterList'
+import { FilterDefinitions, GetFilterListQuery, NamedDates } from '../../types'
+import { useLanguageProvider } from '../../../contexts/Localisation'
+import { USER_ROLES } from '../../data'
+import { usePrefs } from '../../../contexts/SystemPrefs'
 
-export const useApplicationFilters = () => {
+export const useGetFilterDefinitions = () => {
   const { strings } = useLanguageProvider()
 
+  const { preferences } = usePrefs()
+
+  const defaultFilters = preferences?.defaultListFilters || []
```

Comment:
You can leave this for now, but I just wanted to point out that having `[]` as the fallback instead of some hard-coded defaults means we're making more work for ourselves. Whenever we upgrade a server, Jess is going to ask "why are the default filters not showing up?", and then we're going to have to go through every instance and manually add "defaultFilters" to their prefs file.

If you're happy to do that, that's fine, I was just trying to avoid having to re-edit existing prefs files unless they needed to be different to the agreed "standard".